.travis.yml

```diff
@@ -2,36 +2,44 @@ os: osx
-  - bash <(curl -s https://codecov.io/bash) -cF ios -J 'SwifterSwift'
-  - xcodebuild clean build test -project "$PROJECT" -scheme "$TVOS_SCHEME" -destination "$TVOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-tvos.json" xcpretty -f `xcpretty-json-formatter`
-  - bash <(curl -s https://codecov.io/bash) -cF tvos -J 'SwifterSwift'
-  - xcodebuild clean build test -project "$PROJECT" -scheme "$MACOS_SCHEME" -destination "$MACOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-macos.json" xcpretty -f `xcpretty-json-formatter`
-  - bash <(curl -s https://codecov.io/bash) -cF osx -J 'SwifterSwift'
-  - xcodebuild clean build -project "$PROJECT" -scheme "$WATCHOS_SCHEME" -destination "$WATCHOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-watchos.json" xcpretty -f `xcpretty-json-formatter`
-  - bundle exec danger --verbose
-  - bundle exec pod lib lint --skip-tests --allow-warnings
+jobs:
+  include:
+    - name: Darwin 
+      env:
+        - PROJECT='SwifterSwift.xcodeproj'
+        - IOS_SCHEME='SwifterSwift-iOS'
+        - TVOS_SCHEME='SwifterSwift-tvOS'
+        - WATCHOS_SCHEME='SwifterSwift-watchOS'
+        - MACOS_SCHEME='SwifterSwift-macOS'
+        - IOS_DESTINATION='platform=iOS Simulator,name=iPhone X'
+        - TVOS_DESTINATION='platform=tvOS Simulator,name=Apple TV 4K (at 1080p)'
+        - WATCHOS_DESTINATION='name=Apple Watch - 42mm'
+        - MACOS_DESTINATION='platform=OS X'
+      before_install:
+        - bundle install
+        - brew update
+        - brew outdated xctool || brew upgrade xctool
+      script:
+        - set -o pipefail
+        - swiftlint version #cheking the version installed
+        - swift --version
+        - xcodebuild clean build test -project "$PROJECT" -scheme "$IOS_SCHEME" -destination "$IOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-ios.json" xcpretty -f `xcpretty-json-formatter`
+        - bash <(curl -s https://codecov.io/bash) -cF ios -J 'SwifterSwift'
+        - xcodebuild clean build test -project "$PROJECT" -scheme "$TVOS_SCHEME" -destination "$TVOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-tvos.json" xcpretty -f `xcpretty-json-formatter`
+        - bash <(curl -s https://codecov.io/bash) -cF tvos -J 'SwifterSwift'
+        - xcodebuild clean build test -project "$PROJECT" -scheme "$MACOS_SCHEME" -destination "$MACOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-macos.json" xcpretty -f `xcpretty-json-formatter`
+        - bash <(curl -s https://codecov.io/bash) -cF osx -J 'SwifterSwift'
+        - xcodebuild clean build -project "$PROJECT" -scheme "$WATCHOS_SCHEME" -destination "$WATCHOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-watchos.json" xcpretty -f `xcpretty-json-formatter`
+        - bundle exec danger --verbose
+        - bundle exec pod lib lint --skip-tests --allow-warnings
+    - name: Linux 
+      env: SWIFT_VERSION=5.0
+      os: linux
```

Comment:
Does this mean that the tests only run on Linux?

Reply:
nope, they run on both Darwin and Linux :)